docs/guides.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: What is Battlesnake?
 sidebar_position: 0
+slug: /
+displayed_sidebar: guides
```

Comment:
Related to the above change, since this is NOT nested under `guides` anymore we opt into its sidebar explicitly

If we remove the above `slug` and move this file back to its original location this will be removed too!